nixos/modules/services/web-apps/ihatemoney/default.nix

```diff
@@ -132,6 +132,9 @@ in
             chdir = "${pkg}/${pkg.pythonModule.sitePackages}/ihatemoney";
             env = [ "IHATEMONEY_SETTINGS_FILE_PATH=${configFile}" ];
             pythonPackages = self: [ self.ihatemoney ];
+            # workaround for https://github.com/unbit/uwsgi/issues/1126
+            # probably fixed in uwsgi 2.0.19 https://github.com/unbit/uwsgi/commit/d0444450a195f218456a2149d9c2066afa0ae52a
```

Comment:
Released 2 days ago! https://github.com/unbit/uwsgi/releases

Probably better to make that update first & remove this workaround, since otherwise it will likely stick around forever.

Reply:
Do you want me to update uwsgi in this PR ?

Reply:
Either that, or in its own PR (and ping me for review), or hope that somebody else does it soon ;)